.github/workflows/ci.yaml

```diff
@@ -2,9 +2,9 @@ name: CI build
 
 on:
   push:
-    branches: [ main, v1 ]
+    branches: [ main, v1, v2 ]
   pull_request:
-    branches: [ main, v1 ]
+    branches: [ main, v1, v2 ]
```

Comment:
It might be better to use this expression we use in the cryostat, and cryostat-operator repos instead:
https://github.com/cryostatio/cryostat/blob/bb7840b33e288bd40cad2f97df60fea88970c9ce/.github/workflows/ci.yaml#L5-L8